package.json

```diff
@@ -18,7 +18,7 @@
     "@mui/material": "^5.14.18",
     "@mui/material-nextjs": "^5.15.7",
     "aws-amplify": "^6.0.13",
-    "next": "latest",
+    "next": "^14.1.1-canary.56",
```

Comment:
This seems to be the cause of the issue. Could you explain why this is set to a Canary Build?

Reply:
When I tested it the first time, I was running into the "Element type is invalid" error. The error page also said my version of Next.js was out of date. So I upgraded it. I did not realize Canary was a test build. Is there a way to fix this?

Reply:
Setting next to "latest" within package.json should set it to the right version. Let me know if that works or if something else comes up.